_events/2018-11-10-first_workshop.md

```diff
@@ -1,16 +1,138 @@
+
+{% include four_central_image.html
+  image1_path="/images/events/2018-11-10-first_workshop/DSC_6857.jpg"
+  image2_path="/images/events/2018-11-10-first_workshop/DSC_6954.jpg"
+  image3_path="/images/events/2018-11-10-first_workshop/DSC_6853.jpg"
+  image4_path="/images/events/2018-11-10-first_workshop/DSC_6941.jpg"
+%}
+
+We then loaded the MinION with our DNA and launch the sequencing
+
+{% include four_central_image.html
+  image1_path="/images/events/2018-11-10-first_workshop/DSC_6951.jpg"
+  image2_path="/images/events/2018-11-10-first_workshop/DSC_6946.jpg"
+  image3_path="/images/events/2018-11-10-first_workshop/DSC_6973.jpg"
+  image4_path="/images/events/2018-11-10-first_workshop/DSC_6981.jpg"
+%}
+
+It was quite exciting. We had long read until 24Kb, but the quality 
+started to decrease quite quickly (probably not enough DNA) so we stopped
+it earlier.
+
+*The details of the protocol we used can be found in this website: [DNA sequencing protocol]({% link _protocols/beer-dna-sequencing.md %}).*
+
+Milad then analyzed the generated sequences and extracted some *Saccharomyces cerevisae*.
+
+{% include one_central_image.html
+    image_path="/images/events/2018-11-10-first_workshop/analysis_taxonomy.png"
+%}
+
+Very promising results (given we stoped the sequencing)!
+
+## Next steps
+
+Our next challenge will be to simplify the DNA extraction protocol, specifically 
+using simpler and everyday life tools, and ensure its robustness on every location and 
+situation, i.e., the protocol has to work in a specialised lab as well as in a high 
+school chemistry/biology labs. To overcome this challenge, we will collect ideas from 
+Do-It-Yourself biology projects.
+
+We will make special efforts in making the protocols accessible and understandable 
+by everyone, with graphics supporting each step and explaining their importance and 
+their execution. 
+
+For the data analysis, a dedicated workflow will be available via [Galaxy](). 
+This open source, web-based platform makes reproducible computational biology and 
+bioinformatics accessible to anyone via a web-browser. 
+
+A dedicated Galaxy server ([streetscience.usegalaxy.eu](https://streetscience.usegalaxy.eu/)) 
+will be implemented to serve the workflow. In complement, training material for 
+data analysis will be created in cooperation with the [Galaxy Training community](https://galaxyproject.github.io/training-material/) 
+with slides, hands-on material, interactive web-tours suitable to guide our participants 
+from data upload to data analysis. 
+
+As the aim of this project is to share our knowledge with the public, we will 
+organize two-day "DNA extraction, sequencing and data analysis" workshops, 
+one in March at a local biotechnology high school, and one in July for a larger audience. 
```

Comment:
I would not mention time and targets at this point.

Reply:
:+1: